include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -111,7 +111,7 @@ namespace RINGMesh
         GeoModel3D& geomodel_;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make this member variable "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3656)